zerver/management/commands/makemessages.py

```diff
@@ -73,3 +110,83 @@ def my_templatize(src, origin=None):
             trans_real.block_re = old_block_re
             trans_real.templatize = old_templatize
             trans_real.constant_re = old_constant_re
+
+    def extract_strings(self, data):
+        translation_strings = {}
+        for regex in frontend_compiled_regexes:
+            for match in regex.findall(data):
+                translation_strings[match] = ""
+
+        return translation_strings
+
+    def get_translation_strings(self):
+        translation_strings = {}
+        dirname = self.get_template_dir()
+
+        for filename in os.listdir(dirname):
+            if filename.endswith('handlebars'):
+                with open(os.path.join(dirname, filename)) as reader:
+                    data = reader.read()
+                    translation_strings.update(self.extract_strings(data))
+
+        return translation_strings
+
+    def get_template_dir(self):
+        return self.frontend_source
+
+
+    def get_namespace(self):
+        return self.frontend_namespace
+
+    def get_locales(self):
+        locale = self.frontend_locale
+        exclude = self.frontend_exclude
+        process_all = self.frontend_all
+
+        locale_dirs = filter(os.path.isdir, glob.glob('%s/*' % self.default_locale_path))
```

Comment:
Can you write this as `'%s/*' % (self.default_locale_path,))` to avoid any ambiguity with whether the variable is a tuple?